test/demo/iiif.html

```diff
@@ -25,7 +25,7 @@
         // debugMode: true,
         id: "contentDiv",
         prefixUrl: "../../build/openseadragon/images/",
-        tileSources: "http://wellcomelibrary.org/iiif-img/b11768265-0/a6801943-b8b4-4674-908c-7d5b27e70569/info.json",
+        tileSources: "https://iiif.wellcomecollection.org/image/V0021191/info.json",
```

Comment:
What's the purpose of this change?

Reply:
That's great! Definitely good to keep it up to date. I figured it was probably because the old one went away, but I wanted to confirm. 

I would definitely like this change, and I'm fine with having it here in this PR, but if you want to do a separate PR for it, that's okay too.

Reply:
Fixed in #2132